ui/app/i18n/document-upload/locale_en.json

```diff
@@ -17,5 +17,19 @@
     "YEARS_LABEL": "Years",
     "CHOOSE_ANSWER_FROM_DROPDOWN_LABEL": "Choose Answer",
     "DELETE_LABEL": "Delete",
-    "EDIT_LABEL": "Edit"
+    "EDIT_LABEL": "Edit",
+    "VISIT_TYPE_OPHTHALMOLOGY": "Ophthamology",
```

Comment:
@prashantsable1991 there is a small typo on Ophthalmology (missing "l" after 'Ophtha')